force-app/main/default/classes/GiftBatch.cls

```diff
@@ -46,6 +46,9 @@ public inherited sharing class GiftBatch implements IGiftBatch {
     @TestVisible
     private GiftTemplate giftTemplate;
 
+    private final Boolean shouldPayFirstInstallment;
+    private static final String AllowFirstInstallmentPaid = 'AllowFirstInstallment__f';
```

Comment:
From Victor: "It's almost like a fake field, stored in the batch defaults JSON string to be read in the backend to decide when a recurring gift should have it’s first installment paid.
Because I’m storing it in an existing JSON string with a defined structure, I had to follow the existing pattern"